### PR TITLE
BaSP-MR-56: Create activity list/delete table

### PR DESCRIPTION
**Pull request by lorenzomauro59**
I have created the list table with delete functionality with a modal to inform success of delete and styles for the page.

### CODE REVIEW

File: src/Components/Activities/Modals/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.container {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+
```
Comment (AleMac17) [inline src/Components/Activities/Modals/modal.module.css]: We should not leave empty lines

File: src/Components/Activities/Table/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Description</th>
+          <th></th>
+        </tr>
+      </thead>
+
```
Comment (AleMac17) [inline src/Components/Activities/Table/index.js]: we should not leave empty lines

**Review by AleMac17 — CHANGES_REQUESTED** [review]

**Review by AleMac17 — APPROVED** [review]

**Review by LucianoAlarcon — APPROVED** [review]

**Comment by lorenzomauro59** [comment]
[[Activity] List/Delete table](https://trello.com/c/mv287KPM/56-activity-list-delete-table)

**Comment by nazarenapantanetti** [comment]
Tested from master, bugs were found. Link to the report ticket https://trello.com/c/YcXRvhea/89-activity-bug-delete-errors

**Pull request merged**